modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -380,7 +350,7 @@ def swagger_doc
     end
   end
 
-  path '/sc_upload_path' do
+  path '/sc-upload-path' do
```

Comment:
This path name is arbitrary (it is actually auto-generated), so I updated it here to at least use dashes instead of underscores like the other endpoints do.